### DISCUSSION TITLE
360 T7 刷 immortalwrt 具体方法？

### DISCUSSION THREAD

**Discussion by liyiranlab** [immortalwrt/immortalwrt#1257] [Q&A]
之前，刷了H大的uboot和固件，由于H大的uboot有开启NMBM；现在想用 immortalwrt 固件，有两个问题：
1：天灵大佬的immortalwrt 的固件怎么刷，因为现在是H大的uboot和固件，怎么刷天灵大佬的uboot和固件？
2：有uboot吗？可以网页写入吗？

**Comment by liyiranlab** [comment]
https://www.right.com.cn/forum/forum.php?mod=viewthread&tid=8370438&page=1&extra=#pid20225315

**Comment by liyiranlab** [comment]
https://www.right.com.cn/forum/thread-8371712-1-1.html

**Comment by initzhu** [comment]
https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=dc2d4d73939c3d86a8e9d968c5c3462f92771bc6

**Comment by initzhu** [comment]
op和immo的uboot没有web 只能通过串口操作
